templates/platform_landing_zone/variables.tf

```
@@ -6,7 +6,7 @@ variable "starter_locations" {
     error_message = "You must provide at least one starter location region."
   }
   validation {
-    condition     = (length(var.virtual_wan_virtual_hubs) == length(var.starter_locations)) || (length(var.hub_and_spoke_vnet_virtual_networks) == length(var.starter_locations))
+    condition     = var.connectivity_type == "none" || ((length(var.virtual_wan_virtual_hubs) >= length(var.starter_locations)) || (length(var.hub_and_spoke_vnet_virtual_networks) >= length(var.starter_locations)))
```

Sorry, I revered the logic:
```suggestion
    condition     = var.connectivity_type == "none" || ((length(var.virtual_wan_virtual_hubs) <= length(var.starter_locations)) || (length(var.hub_and_spoke_vnet_virtual_networks) <= length(var.starter_locations)))
```